src/main/pftconMod.F90

```diff
@@ -164,6 +164,10 @@ module pftconMod
      real(r8), allocatable :: wood_density  (:)   ! wood density (kg/m3)
      real(r8), allocatable :: crit_onset_gdd_sf(:)! scale factor for crit_onset_gdd
      real(r8), allocatable :: ndays_on(:)         ! number of days to complete leaf onset
+     real(r8), allocatable :: jmaxb0(:)           ! Baseline proportion of nitrogen allocated for electron transport (J)
+     real(r8), allocatable :: jmaxb1(:)           ! Coefficient determining the response of electron transport rate to light availability (-)
+     real(r8), allocatable :: wc2wjb0(:)          ! The baseline ratio of rubisco limited rate vs light limited photosynthetic rate (Wc:Wj) (-)
+     real(r8), allocatable :: r_mort(:)           ! Mortality rate (1/year)
```

Comment:
@olyson I understand that the code works as written. I will ask this anyway in case it's relevant and worth considering:

As a general rule we try to place here (pftconMod) the params that appear in multiple places in the code. On the other hand, we try to place params that appear in a single module in the module where they appear.

Reply:
Right, I started down the path of placing the parameters within LunaMod, but realized we don't have the infrastructure in there to do that (e.g., as is done in PhotosynthesisMod.F90 with the photo_params_type and subroutines allocParams, cleanParams, etc.).  Much easier to put them in pftconMod.F90 instead of putting all of that infrastructure in, and I wasn't sure what the real value of all that was anyway.  But if there is consensus to do it that way, I can do it.

Reply:
I am also often motivated by efficiency in code development, so I will let @ekluzek have final say about this.

Reply:
Thinking about it some more, now that my head is in parameter space, I think I'll just put the work in now to have the parameters read in in LunaMod.F90.

Reply:
And CNGapMortalityMod.F90 (for r_mort).